bin/juttle-engine

```diff
@@ -7,7 +7,7 @@ var engine = require('..');
 
 var defaults = {
     'port': 8080,
-    'root': '/',
+    'root': process.cwd(),
```

Comment:
Hmm... the benefit of using __dirname is that the examples will work when invoking juttle-engine from a path (i.e. if installed in /usr/local/lib/node_modules or in ~/node_modules). This would work more reliably to run the demo programs as a new user.

The benefit of cwd is that it's more intuitive if the user has juttle files in their home directory, they could just cd there and run juttle-engine. This is more what I would expect when running it myself or in production.

Given that neither option is universally good, I'm inclined towards the time-to-wow and making it __dirname like @go-oleg suggested.